packet/flatcar-linux/kubernetes/workers/variables.tf

```diff
@@ -116,6 +116,12 @@ variable "setup_raid_ssd" {
   default     = "false"
 }
 
+variable "setup_raid_ssd_block" {
+  description = "Keep the SSD RAID as block device. Valid values: \"true\", \"false\""
```

Comment:
"If true, file system will be created on raid device and will be mounted on `/foo`"?

Reply:
done, testing it now!